recipes/pyvisa-py/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "PyVISA-py" %}
+{% set version = "0.2" %}
+{% set sha256 = "4e77ac0302f13f22e003d9be07ede371ead3fd672a17d9a963e24e8babc29ad7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch_python: True
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyvisa >=1.8
+
+  run:
+    - python
+    - pyvisa >=1.8
+    - pyserial # optional
+    #- pyusb # optional
+
+test:
+  imports:
+    - pyvisa
+
+  commands:
+    - python -c "import pyvisa.testsuite as ts; ts.run()"
+
+about:
+  home: https://github.com/hgrecco/pyvisa-py
+  license: MIT
```

Comment:
Please add `license_file: LICENSE` so as to include the license in the `conda` package.